.nvmrc

```diff
@@ -0,0 +1 @@
+12.14.1
```

Comment:
Is there any reason this can't just be `12`? As is, it won't let me commit when I'm using NodeJS `12.21.0`.

Reply:
The intention is that by using the exact same version here and on CI, there are not any unexpected differences between our usages.